docker/replay.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# Copyright(C) 2021, Stamus Networks
+# Written by Raphaël Brogat <rbrogat@stamus-networks.com>
+#
+# This script comes with ABSOLUTELY NO WARRANTY!
+#
+# GPLv3 <http://www.gnu.org/licenses/>.
+
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+function Help(){
+  # Display Help
+  { echo
+    echo "Pcap replaying script through Suricata"
+    echo
+    echo -e "\t Syntax: replay.sh [OPTIONS] <path>"
+    echo
+    echo "<path>"
+    echo "    Path to the pcap file to replay. If <path> specifies a directory, all files in that directory"
+    echo "    will be processed in order of modified time maintaining flow state between files."
+    echo
+    echo "OPTIONS"
+    echo " -h"
+    echo "   Display help"
+    echo " -c"
+    echo "   Cleanup : remove all previous data from elasticsearch and suricata."
+
+  } | fmt
+}
+
+function Cleanup(){
+    ${BASEDIR}/cleanup.sh
+}
+
+if [ $# -lt 1 ]; then
+    Help
+    exit
+fi
+
+############################################################
+# Process the input options. Add options as needed.        #
+############################################################
+# Get the options
+while getopts ":hc" option; do
+   case $option in
+      h) # display Help
+         Help
+         exit;;
+      c) # Cleanup
+         Cleanup;;
+      \?) # Invalid option
+         echo "Error: Invalid option"
+         exit;;
+   esac
+done
+
+# The path is the last argument
+for i in $@; do :; done
+HOST_PATH="$i"
+
+if [[ -d ${HOST_PATH} ]]; then
+    LOCAL_PATH="/replay"
+else
+    LOCAL_PATH="/replay/file.pcap"
+fi
+
+docker run --name suricata-replay --rm -it \
+--cap-add=net_admin --cap-add=sys_nice \
+-v ${BASEDIR}/containers-data/suricata/logs:/var/log/suricata \
+-v selks_suricata-rules:/etc/suricata/rules \
+-v ${BASEDIR}/containers-data/suricata/etc:/etc/suricata \
+-v ${HOST_PATH}:${LOCAL_PATH} \
+jasonish/suricata:master -r ${LOCAL_PATH} --runmode autofp -l /var/log/suricata --set sensor-name=suricata-replay
```

Comment:
add in please :
1- `-k none` 
2 - use `--runmode=single` by default
3 - open an issue to be able to supply if desired `--runmode=autofp`

Reply:
Lets try the following:
1- run in unix-socket mode   
2- pass the pcap via unix socket  (continuous mode will be added later as a possibility, lets use the default/easy mode first )

Separate issue:
if the cleanup script is used, make sure we send SIGHUP signal to the Suricata process

Reply:
> add in please : 1- `-k none` 2 - use `--runmode=single` by default 3 - open an issue to be able to supply if desired `--runmode=autofp`

all fixed

Reply:
> Separate issue: if the cleanup script is used, make sure we send SIGHUP signal to the Suricata process

done.

Reply:
> Lets try the following: 1- run in unix-socket mode 2- pass the pcap via unix socket (continuous mode will be added later as a possibility, lets use the default/easy mode first )

Let's make this a separate issue so that we can merge as it is for now ?